src/main/java/com/google/devtools/build/lib/bazel/bzlmod/ModuleFileFunction.java

```diff
@@ -640,7 +646,8 @@ private GetModuleFileResult getModuleFile(
       }
       RootedPath moduleFilePath =
           RootedPath.toRootedPath(
-              Root.fromPath(repoDir.getPath()), LabelConstants.MODULE_DOT_BAZEL_FILE_NAME);
+              Root.fromPath(((Success) repoDir).getPath()),
```

Comment:
Is it clear that this cast is safe? Could you add a comment explaining why?

Reply:
it's indeed safe -- added a comment.